<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BattleLocalGame</class>
 <widget class="QWidget" name="BattleLocalGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BattleShip Local 2V2 Game</string>
  </property>
  <widget class="QLabel" name="GameLogo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BattleShip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RandomButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Random Board</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CustomButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Custom Board</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="PlayerGameBoard">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
